app/__init__.py

```diff
@@ -5,8 +5,8 @@
 def create_app():
     app = Flask(__name__)
     app.config['DEBUG'] = True
-    
+
     # Register the blueprints for endpoints in app instance
-    app.register_blueprint(controllers.get_index_bp)
+    app.register_blueprint(controllers.get_about_page)
```

Comment:
@onlyVishesh I don't think you addressed my previous comments from the other PR, please don't change anything and add your code accordingly. PTAL at your prev PR and make changes accordingly. I am holding on review process untill you do. 

Also try to address all the comments in one go and then assign the reviewers. Thanks

Reply:
ok sir